extensions/amp-live-list/0.1/test/test-live-list-manager.js

```diff
@@ -137,6 +140,17 @@ describe('LiveListManager', () => {
     });
   });
 
+  it('should get the amp_latest_update_time on doc ready', () => {
```

Comment:
Not sure if it's feasible, but it would be better to test where this private value is used than where it is filled.
In this case, a more robust test would be to check whether the number is finally in the fetch request URL.